packages/airnode-node/test/fixtures/requests/api-calls.ts

```diff
@@ -26,14 +26,28 @@ export function buildApiCall(params?: Partial<Request<ApiCall>>): Request<ApiCal
   };
 }
 
-export function buildSubmittableApiCall(params?: Partial<Request<ApiCall>>): Request<ApiCall> {
+export function buildSuccessfulApiCall(
+  params?: Partial<Request<ApiCallWithResponse>> & { success?: true }
```

Comment:
You need this, because these properties are only defined in successful request. If you omit it, Typescript will complain that `success` could be `false` and you could be missing required `errorMessage` property.

Reply:
Why isn't it `params?: Partial<Request<RegularApiCallSuccessResponse>>` then?

Reply:
Good point. I'll replace.